internal/backends/winit/glwindow.rs

```diff
@@ -510,54 +513,74 @@ impl<Renderer: WinitCompatibleRenderer + 'static> WindowAdapterSealed for GLWind
                 }
             }
 
-            let window_builder = if std::env::var("SLINT_FULLSCREEN").is_ok() {
-                window_builder.with_fullscreen(Some(winit::window::Fullscreen::Borderless(None)))
+            if std::env::var("SLINT_FULLSCREEN").is_ok() {
+                window_builder = window_builder
+                    .with_fullscreen(Some(winit::window::Fullscreen::Borderless(None)));
             } else {
-                if layout_info_h.min >= 1. as Coord || layout_info_v.min >= 1. as Coord {
-                    window_builder = window_builder.with_min_inner_size(into_size(
-                        winit::dpi::LogicalSize::new(layout_info_h.min, layout_info_v.min),
-                    ))
+                let min_inner_size =
+                    if layout_info_h.min >= 1. as Coord || layout_info_v.min >= 1. as Coord {
+                        Some(winit::dpi::LogicalSize::new(layout_info_h.min, layout_info_v.min))
```

Comment:
How about merging both in a helper function?
`fn min_max_sizes(h: LayoutInfo, v: LayoutInfo) -> (Option<winit::dpi::LogicalSize>, Option<winit::dpi::LogicalSize>)`

Reply:
Nice! I'll do that.